src/admin-views/widgets/components/dropdown.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Admin View: Widget Dropdown Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/dropdown.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @var string  $label   Label for the checkbox.
+ * @var string  $value   Value for the checkbox.
+ * @var string  $id      ID of the checkbox.
+ * @var string  $name    Name attribute for the checkbox.
+ * @var boolean $checked Whether the checkbox is checked or not.
+ *
+ * @version TBD
+ *
+ */
+
+?>
+<p
+		class="tribe-filter-bar-c-dropdown tribe-common-form-control-dropdown"
+>
+	<label
+			class="tribe-common-form-control-dropdown__label"
+			for="<?php echo esc_attr( $id ); ?>"
+	>
+		<?php echo esc_html( $label ); ?>
+	</label>
+	<select
+			id="<?php echo esc_attr( $id ); ?>"
+			name="<?php echo esc_attr( $name ); ?>"
+			class="tribe-common-form-control-dropdown__input widefat"
+	>
+		<?php foreach ( $options as $option ) {
+			?>
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: ec8d1433c3294fe7a38ba09300168629</sub>